inc/saiport.h

```diff
@@ -438,7 +438,7 @@ typedef enum _sai_port_attr_t
      * @brief Media Type
      *
      * @type sai_port_media_type_t
-     * @flags CREATE_ONLY
+     * @flags CREATE_AND_SET
```

Comment:
Should that operation require to recreate port again ? since if they are hot swap then other (not all?) 
 attributes on port should be create_and_set also